molecule/default/tests/test_create_snapshot_on_instance.py

```diff
@@ -32,23 +33,19 @@ def test_create_snapshot_of_an_instance(host):
         "network_name": private_net,
     }
 
-    utils.create_instance(data_image, host)
-    assert (utils.verify_asset_in_list('server', instance_name, host))
+    helpers.create_instance(data_image, host)
+    sleep(30)  # TODO: will find out a better way to avoid implicit sleep.
+    assert (helpers.asset_is_in_the_list('server', instance_name, host))
```

Comment:
Are the sleep call and the `asset_is_in_the_list` call necessary given the retry nature of the `get_expected_value` call below?